doc/nrf/releases_and_maturity/migration_guides.rst

```diff
@@ -9,10 +9,16 @@ Migration guides are also provided for major functionality updates.
 .. note::
     |migration_contact_devzone|
 
-.. toctree::
-   :maxdepth: 1
-   :glob:
-   :reversed:
-   :caption: Subpages:
+* `Migrating from multi-image builds to sysbuild`_
+* `Migrating to the current hardware model`_
+* `Migration guide for nRF Connect SDK v2.7.0`_
```

Comment:
Missing entry for 2.8?

Reply:
Since the migration guide is still in the draft state, I didn't add it to the list. 
I think it make sense to add it, though its in the draft state. added.